packages-web/maps/package.json

```diff
@@ -15,8 +15,8 @@
         "test": "npm run test:unit",
         "test:unit": "..\"/../node_modules/.bin/jest\" --config ../../scripts/test/jest.web.config.js",
         "pretest:e2e": "..\"/../node_modules/.bin/cross-env\" URL=https://leafletmaps.mxapps.io ../../scripts/test/projectup.sh",
-        "test:e2e": "..\"/../node_modules/.bin/cross-env\" URL=https://leafletmaps.mxapps.io/ .\"/node_modules/.bin/utils-react-widgets\" test:e2e",
-        "test:e2e:dev": "..\"/../node_modules/.bin/cross-env\" DEBUG=true .\"/node_modules/.bin/utils-react-widgets\" test:e2e",
+        "test:e2e": "..\"/../node_modules/.bin/cross-env\" URL=https://leafletmaps.mxapps.io ../../node_modules/.bin/wdio tests/e2e/wdio.conf.js",
```

Comment:
Please create a new PR changing this config to global wdio.conf

Reply:
done